qiskit/algorithms/optimizers/optimizer.py

```diff
@@ -212,6 +214,11 @@ def optimize(
             )
         pass
 
+    @property
+    def settings(self) -> Dict[str, Any]:
```

Comment:
Similar to the other PR, can we make this an abstract method so new child class will implement it?

Reply:
👍🏻